app/config/collections.php

```diff
@@ -3382,7 +3382,7 @@
                 '$id' => ID::custom('providerCommitMessage'),
                 'type' => Database::VAR_STRING,
                 'signed' => true,
-                'size' => Database::LENGTH_KEY,
+                'size' => 65535,
```

Comment:
For this change to apply we also need a patch to run against the production env. You can check with @christyjacob4 - we probably have something for this. Also, this change forces a migration so should go to 1.7 branch.